pcdsdevices/ipm.py

```diff
@@ -5,6 +5,7 @@
 from ophyd.device import Device
 from ophyd.device import FormattedComponent as FCpt
 from ophyd.signal import EpicsSignal, EpicsSignalRO
+from ophyd.status import wait as status_wait
```

Comment:
This is actually deprecated in favor of `Status.wait`

Reply:
ok good to know, thank you